src/screens/subscriber/components/TransactionsTable.js

```diff
@@ -24,13 +24,25 @@ const TransactionsTable = ({
     'balance'
   ];
   const headerVisible = showHeader === undefined ? true : showHeader;
+
+  const evalWidth = element => {
+    switch (element) {
+      case 'description':
+        return 200;
+      case 'date':
+        return 150;
+      default:
+        return undefined;
```

Comment:
```suggestion
        return -1;
```

I would return an integer value here just to keep things consistent

Reply:
Returning undefined seems to be used to indicate width: "auto" of the table whereas returning some integer restricts or expands to that width